simtools/applications/generate_corsika_histograms.py

```diff
@@ -235,74 +220,64 @@ def _parse(label, description, usage):
 
     config.parser.add_argument(
         "--event_1D_histograms",
-        help="Arguments from the CORSIKA event header to extract 1D histograms.",
+        help="The keys from the CORSIKA event header to be used for the generation of 1D "
```

Comment:
Is there a place you can point to where someone can see the list of options? Not critical though.

Reply:
ups, just commented above about it. I am not sure it is useful for the user, because from the application itself it would not be possible to access a class attribute